app/views/pipelines/_run_pipeline.html.erb

```diff
@@ -126,6 +126,25 @@
           </div>
         </div>
 
+         <div class='col-5'>
+          <%= form.label :run_enrichment_concurrently, class: 'form-label' do %>
+            Run Enrichment Concurrently
```

Comment:
Little thing but I think this should be plural as you can have multiple enrichments